cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -113,9 +115,10 @@ func NewCMServer() *CMServer {
 		ConcurrentRCSyncs:                 5,
 		ConcurrentDSCSyncs:                2,
 		ConcurrentJobSyncs:                5,
+		ConcurrentResourceQuotaSyncs:      5,
 		ServiceSyncPeriod:                 5 * time.Minute,
 		NodeSyncPeriod:                    10 * time.Second,
-		ResourceQuotaSyncPeriod:           10 * time.Second,
+		ResourceQuotaSyncPeriod:           10 * time.Minute,
```

Comment:
Obviously the summary and the code got out of sync.

I will drop this down to 5 minutes because that feels appropriate to me.